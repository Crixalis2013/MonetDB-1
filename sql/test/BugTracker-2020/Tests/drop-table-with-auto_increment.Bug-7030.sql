create local temp table "depend_count" ("a" bigint) on commit preserve rows;
insert into "depend_count" values ((select count(*) from dependencies, sequences));

create schema s1;
create table s1.t (i int not null auto_increment);
drop table s1.t;
drop schema s1;

create schema "a""a";
create table "a""a"."nasty"(a int auto_increment);
insert into "a""a"."nasty";
select a from "a""a"."nasty";
	-- 1
drop table "a""a"."nasty";
drop schema "a""a";

select cast(count(*) - (select "a" from "depend_count") as bigint) from dependencies, sequences;
	-- the number of dependencies and sequences shouldn't increase
drop table "depend_count";
