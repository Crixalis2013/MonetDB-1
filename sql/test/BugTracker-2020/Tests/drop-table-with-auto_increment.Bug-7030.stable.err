stderr of test 'drop-table-with-auto_increment.Bug-7030` in directory 'sql/test/BugTracker-2020` itself:


# 12:21:18 >  
# 12:21:18 >  "mclient" "-lsql" "-ftest" "-tnone" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-868236" "--port=30843"
# 12:21:18 >  


# 12:21:18 >  
# 12:21:18 >  "Done."
# 12:21:18 >  

