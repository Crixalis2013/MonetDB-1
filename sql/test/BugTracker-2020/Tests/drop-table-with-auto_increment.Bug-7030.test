statement ok
create local temp table "depend_count" ("a" bigint) on commit preserve rows

statement ok
insert into "depend_count" values ((select count(*) from dependencies, sequences))

statement ok
create schema s1

statement ok
create table s1.t (i int not null auto_increment)

statement ok
drop table s1.t

statement ok
drop schema s1

query I nosort
select cast(count(*) - (select "a" from "depend_count") as bigint) from dependencies, sequences
----
0

statement ok
drop table "depend_count"

