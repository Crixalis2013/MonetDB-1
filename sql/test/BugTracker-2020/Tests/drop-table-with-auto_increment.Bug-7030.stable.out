stdout of test 'drop-table-with-auto_increment.Bug-7030` in directory 'sql/test/BugTracker-2020` itself:


# 12:21:18 >  
# 12:21:18 >  "mclient" "-lsql" "-ftest" "-tnone" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-868236" "--port=30843"
# 12:21:18 >  

#create local temp table "depend_count" ("a" bigint) on commit preserve rows;
#insert into "depend_count" values ((select count(*) from dependencies, sequences));
[ 1	]
#create schema s1;
#create table s1.t (i int not null auto_increment);
#drop table s1.t;
#drop schema s1;
#create schema "a""a";
#create table "a""a"."nasty"(a int auto_increment);
#insert into "a""a"."nasty";
[ 1	]
#select a from "a""a"."nasty";
% a\"a.nasty # table_name
% a # name
% int # type
% 1 # length
[ 1	]
#drop table "a""a"."nasty";
#drop schema "a""a";
#select cast(count(*) - (select "a" from "depend_count") as bigint) from dependencies, sequences;
% .%3 # table_name
% %3 # name
% bigint # type
% 1 # length
[ 0	]
#drop table "depend_count";

# 12:21:18 >  
# 12:21:18 >  "Done."
# 12:21:18 >  

